src/backends/sqlite3/session.cpp

```diff
@@ -126,6 +130,12 @@ sqlite3_session_backend::sqlite3_session_backend(
         execude_hardcoded(conn_, query.c_str(), errMsg.c_str());
     }
 
+    if (!foreignKeys.empty())
+    {
+        std::string const query("pragma foreign_keys=" + foreignKeys);
+        execude_hardcoded(conn_, query.c_str(), "Attempt to set foreign_keys pragma failed");
```

Comment:
I think it would be useful to show the value here, e.g.
```suggestion
        execude_hardcoded(conn_, query.c_str(), "Executing " + query + " failed");
```